spec/requests/api/v1/users/sign_in_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'User login', type: :request do
+  before do
+    post api_v1_user_session_path, params: user_params, as: :json
+  end
+
+  context 'when user credentials are correct' do
+    let(:password) { Faker::Internet.password }
+    let(:user) { User.create!(attributes_for(:user, password: password)) }
```

Comment:
We should use FactoryBot for creating test data

Reply:
The problem of using FactoryBot in this case, is that it won't encrypt the password when creating the record